pii_recognition/evaluation/pipeline.py

```diff
@@ -0,0 +1,48 @@
+from typing import Dict, List, Tuple, Type
+
+from pakkr import returns
+
+from pii_recognition.evaluation.model_evaluator import ModelEvaluator
+from pii_recognition.recognisers.entity_recogniser import EntityRecogniser
+
+
+@returns(Type[EntityRecogniser])
```

Comment:
why not just `EntityRecogniser`?

Reply:
`EntityRecogniser` represents instances, `Type[EntityRecogniser]` represents the class. `mypy` will complaint this kind of error.